.github/workflows/inductor-periodic.yml

```diff
@@ -67,6 +67,47 @@ jobs:
       test-matrix: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-periodic-dynamo-benchmarks-build.outputs.test-matrix }}
     secrets: inherit
 
+  linux-focal-rocm6_3-py3_10-periodic-dynamo-benchmarks-build:
```

Comment:
Sorry, I should have take a closer look at the PR sooner, but the new benchmark jobs needs to be added to https://github.com/pytorch/pytorch/blob/main/.github/workflows/inductor-perf-test-nightly.yml instead.  There is nothing wrong with the current batch of jobs in inductor-periodic, but they only run accuracy test for torch.compile vs eager , which are not those powering https://hud.pytorch.org/benchmark/compilers dashboard.

Let me update the PR a bit

Reply:
Here is my test PR https://github.com/pytorch/pytorch/pull/145546 and the test run is at https://github.com/pytorch/pytorch/actions/runs/12939147191